migrations/2_deploy_organization.js

```diff
@@ -30,50 +31,48 @@ module.exports = async (deployer) => {
   const orgName = "Genesis";
   const tokenName = "Gen";
   const tokenSymbol = "GEN";
-  const founders = [web3.eth.accounts[0]];
-  const initRepInWei = [web3.toWei(10)];
-  const initTokenInWei = [web3.toWei(1000)];
   const orgNativeTokenFee = 0;
   const defaultVotingMachineParams = {
     preBoostedVoteRequiredPercentage: 50,
-    preBoostedVotePeriodLimit: 60,
-    boostedVotePeriodLimit: 60,
-    thresholdConstA: 1,
-    thresholdConstB: 1,
+    preBoostedVotePeriodLimit: 5184000, // 2 months
+    boostedVotePeriodLimit: 604800, // 1 week
+    thresholdConstA: 1, // TODO: how many proposals can be boosted at a time. want about 5, have to read arc to figure out
+    thresholdConstB: 1, // TODO: exponent of how hard to get the next one
     minimumStakingFee: 0,
-    quietEndingPeriod: 0,
-    proposingRepRewardConstA: 1,
-    proposingRepRewardConstB: 1,
-    stakerFeeRatioForVoters: 1,
-    votersReputationLossRatio: 10,
-    votersGainRepRatioFromLostRep: 80,
+    quietEndingPeriod: 7200, // Two hours
+    proposingRepRewardConstA: 5, // baseline rep rewarded TODO: good for now but Adam going to look up what this should be
```

Comment:
I would remove personal names .
It should be 

> baseline rep rewarded TODO: figure out the real default value.

Reply:
these are not complete